<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Registro</class>
 <widget class="QDialog" name="Registro">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>648</width>
    <height>405</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <widget class="QPushButton" name="boton_nuevo">
   <property name="geometry">
    <rect>
     <x>240</x>
     <y>350</y>
     <width>111</width>
     <height>41</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Century Gothic</family>
     <pointsize>11</pointsize>
    </font>
   </property>
   <property name="text">
    <string>Crear</string>
   </property>
  </widget>
  <widget class="QLabel" name="registro">
   <property name="geometry">
    <rect>
     <x>250</x>
     <y>20</y>
     <width>131</width>
     <height>41</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Century Gothic</family>
     <pointsize>20</pointsize>
    </font>
   </property>
   <property name="text">
    <string>Registro:</string>
   </property>
  </widget>
  <widget class="QWidget" name="gridLayoutWidget">
   <property name="geometry">
    <rect>
     <x>70</x>
     <y>66</y>
     <width>491</width>
     <height>251</height>
    </rect>
   </property>
   <layout class="QGridLayout" name="gridLayout">
    <item row="0" column="0">
     <widget class="QLabel" name="label_correo">
      <property name="font">
       <font>
        <family>Century Gothic</family>
        <pointsize>11</pointsize>
       </font>
      </property>
      <property name="text">
       <string>Correo electrónico:</string>
      </property>
     </widget>
    </item>
    <item row="1" column="1">
     <widget class="QLabel" name="label_validacorreo">
      <property name="font">
       <font>
        <family>Century Gothic</family>
        <pointsize>8</pointsize>
       </font>
      </property>
      <property name="text">
       <string>Correo inválido</string>
      </property>
     </widget>
    </item>
    <item row="2" column="0">
     <widget class="QLabel" name="label_password">
      <property name="font">
       <font>
        <family>Century Gothic</family>
        <pointsize>11</pointsize>
       </font>
      </property>
      <property name="text">
       <string>Contraseña:</string>
      </property>
     </widget>
    </item>
    <item row="1" column="0">
     <widget class="QLineEdit" name="texto_correonuevo">
      <property name="font">
       <font>
        <pointsize>12</pointsize>
       </font>
      </property>
     </widget>
    </item>
    <item row="3" column="0">
     <widget class="QLineEdit" name="texto_passwnueva">
      <property name="font">
       <font>
        <pointsize>12</pointsize>
       </font>
      </property>
      <property name="echoMode">
       <enum>QLineEdit::Password</enum>
      </property>
     </widget>
    </item>
    <item row="4" column="0">
     <widget class="QLabel" name="label_confirmar">
      <property name="font">
       <font>
        <family>Century Gothic</family>
        <pointsize>11</pointsize>
       </font>
      </property>
      <property name="text">
       <string>Confirmar contraseña:</string>
      </property>
     </widget>
    </item>
    <item row="3" column="1">
     <widget class="QLabel" name="label_validapasw">
      <property name="font">
       <font>
        <family>Century Gothic</family>
        <pointsize>8</pointsize>
       </font>
      </property>
      <property name="text">
       <string>Contraseña incorrecta</string>
      </property>
     </widget>
    </item>
    <item row="5" column="0">
     <widget class="QLineEdit" name="texto_confirmar">
      <property name="font">
       <font>
        <pointsize>12</pointsize>
       </font>
      </property>
      <property name="echoMode">
       <enum>QLineEdit::Password</enum>
      </property>
     </widget>
    </item>
    <item row="5" column="1">
     <widget class="QLabel" name="label_coincidir">
      <property name="font">
       <font>
        <family>Century Gothic</family>
        <pointsize>8</pointsize>
       </font>
      </property>
      <property name="text">
       <string>la contraseña no coincide</string>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
